packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -296,6 +303,18 @@ export default class ImageEdit implements EditorPlugin {
 
             // Get initial edit info
             this.editInfo = getEditInfoFromImage(image);
+            if (
+                this.image.style.maxWidth === 'initial' &&
+                this.image.style.height !== 'initial' &&
```

Comment:
here you check height !== initial, but for style.width you only do null check. Any reason not doing the same check?

Actually checking for initial is not always accurate, its value can also be "auto" or "inherit".

I think what you can try is to parse the value to be a number, then check if the result is a valid number, but not NaN:

```ts
function isFixedNumberValue(v: string) {
  return parseInt(v) !== NaN;
}
```